vendor/git.fd.io/govpp.git/cmd/binapi-generator/generator.go

```diff
@@ -540,7 +540,8 @@ func translateVppType(ctx *context, vppType string, isArray bool) string {
 		return typ
 	}
 
-	panic(fmt.Sprintf("Unknown VPP type %s", vppType))
+	//panic(fmt.Sprintf("Unknown VPP type %s", vppType))
+	return "uint8"
```

Comment:
It seems like `enums` were finally introduced in VPP's API. This will need to be added to GoVPP as well.